tests/integration/security/rbac/v2/basic/basic_test.go

```diff
@@ -34,15 +34,16 @@ import (
 )
 
 const (
-	rbacClusterConfigTmpl = "testdata/istio-clusterrbacconfig.yaml.tmpl"
-	rbacV2RulesTmpl       = "testdata/istio-rbac-v2-rules.yaml.tmpl"
+	rbacClusterConfigTmpl   = "testdata/istio-clusterrbacconfig.yaml.tmpl"
+	rbacV2RulesTmpl         = "testdata/istio-rbac-v2-rules.yaml.tmpl"
+	extendedRbacV2RulesTmpl = "testdata/istio-extended-rbac-v2-rules.yaml.tmpl"
```

Comment:
`extendedRbacV2RulesTmpl` is only used in `extended_test.go`, why not define it there?

Reply:
Oh, these files were meant to be defined in `main_test.go`, not sure why they ended in `basic_test.go`. I will put RbacClusterConfig in `main_test.go` and other files to their test files.